### PR TITLE
Add flake retries to eks correctness jobs

### PR DESCRIPTION
**Pull request by shyamjvs**
/cc @gyuho

### CODE REVIEW

File: config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ periodics:
       - --gce-ssh=
       - --extract=ci/latest-1.11
       - --ginkgo-parallel=30
-      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
+      - --test_args=--ginkgo.flakeAttempts=2 --ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml]: oh no! 😛 cc @dims :-)
https://github.com/kubernetes/kubernetes/issues/68091#issuecomment-455787724

Comment (shyamjvs) [reply config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml]: Lol

**Comment by krzyzacy** [comment]
it defeats the purpose for a periodic job right?......

**Comment by shyamjvs** [comment]
Why? It could even mean that e2e tests should be more robust :)

**Comment by krzyzacy** [comment]
yeah... technically we want to expose the flake and fix them instead of hiding them with retries, aka, file issues against flakes in k/k instead :-)

**Pull request merged**